File: api-microservice/src/main/java/io/github/brunogabriel/apimicroservice/domain/Pet.java
package io.github.brunogabriel.apimicroservice.domain;

import lombok.Data;

import java.util.List;

/**
 * Created by brunogabriel on 2019-06-17.
 */
@Data
public class Pet {
    public Long id;
    public String clientId;
    public String type;
    public String name;
    public String gait;
    public List<String> allergies;
    public List<String> habits;
    public List<String> preferences;
    public List<String> images;
    public int quantityOfServices;
}

File: DoggieApp/app/src/main/java/io/github/brunogabriel/doggieapp/shared/models/Pet.kt
package io.github.brunogabriel.doggieapp.shared.models

import android.os.Parcelable
import kotlinx.android.parcel.Parcelize

/**
 * Created by brunogabriel on 2019-06-15.
 */
@Parcelize
class Pet(
    val id: Long,
    val clientId: String,
    val type: String,
    val name: String,
    val gait: String?,
    val allergies: List<String>?,
    val habits: List<String>?,
    val preferences: List<String>?,
    val images:  List<String>?,
    val quantityOfServices: Int = 0
) : Parcelable